### PR TITLE
Update required TLS version

### PR DESCRIPTION
**Pull request by jonashrem**
This page states, that TLS 1.1 is required, but for both paypal and repo.magento.com TLS 1.2 is required.

For Paypal, the information is stated here https://devdocs.magento.com/guides/v2.3/install-gde/system-requirements_tls1-2.html 
For repo.magento.com this ssl test: https://www.ssllabs.com/ssltest/analyze.html?d=repo.magento.com&hideResults=on shows, that TLS 1.2 is required as well.

## Purpose of this pull request

This pull request (PR) updates the minimal TLS version required for paypal and magento,

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.3/install-gde/system-requirements-tech.html

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: guides/v2.3/install-gde/system-requirements-tech.md [hunk 1]
```diff
@@ -77,7 +77,7 @@ For more information, see [Required PHP settings]({{ page.baseurl }}/install-gde
 
 * A valid [security certificate](https://glossary.magento.com/security-certificate) is required for HTTPS.
 * Self-signed SSL certificates are not supported.
-* Transport Layer Security (TLS) requirement - PayPal and `repo.magento.com` both require TLS 1.1 or later:
+* Transport Layer Security (TLS) requirement - PayPal and `repo.magento.com` both require TLS 1.2 or later:
```
Comment (rogyar) [inline guides/v2.3/install-gde/system-requirements-tech.md]: Hi @jonashrem. Thank you for the modification, it completely makes sense.
This update is fair for both Magento 2.2.x and Magento 2.3.x. So, I would kindly ask you to provide the change for 2.2 as well. 

Thank you!

Comment (jonashrem) [reply guides/v2.3/install-gde/system-requirements-tech.md]: sure

**Pull request merged**